mds/apis/agency_api/v0_x/vehicles.py

```diff
@@ -142,10 +168,12 @@ def create(self, validated_data):
         )
 
 
-# TODO: these are in the spec but I don't see what it adds.
-# class DeviceEventResponseSerializer(serializers.Serializer):
-#     device_id = serializers.UUIDField()
-#     status = serializers.ChoiceField(choices=enums.DEVICE_STATUS_CHOICES)
+class DeviceEventResponseSerializer(serializers.Serializer):
+    device_id = serializers.UUIDField()
+    status = serializers.ChoiceField(
+        source="device.latest_event.updated_status",
```

Comment:
The input data for this serialiser is the event itself. cf. L.265
So, the source should be simply `updated_status` : the code here goes from the event, gets the `device` (1 query), get the `latest_event` (1 query), then returns the status.